### PR TITLE
Inline form in contacts

### LINKED ISSUE #31: Inline form in contacts
Should be inline with a combobox to select link type + icons

Also should check that the profile gets updpated

Also can we fix the trigger link icon to be centered please
image

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

Resolve #31

### CODE REVIEW

**Comment by tsumo** [comment]
Rebase please

File: src/components/ services/ServiceForm.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+export const ServiceForm: React.FC<ServicesFormProps> = ({ user }) => {
+    const utils = trpc.useContext();
+    const [search, setSearch] = useState('');
+    const [selectedService, setSelectedService] = useState<ExternalService>();
+    const [serviceId, setServiceId] = useState('');
+    const addService = trpc.services.addToUser.useMutation();
+    const servicesListQuery = trpc.services.getList.useQuery({ search, take: 10 });
+
+    const onReset = () => {
```
Comment (tsumo) [inline src/components/ services/ServiceForm.tsx:47]: Service id should also be reset here

File: src/components/ services/ServiceForm.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    flex-wrap: nowrap;
+    display: flex;
+    align-items: center;
+`;
+
+interface ServicesFormProps {
+    user: User;
+}
+
+export const ServiceForm: React.FC<ServicesFormProps> = ({ user }) => {
```
Comment (tsumo) [inline src/components/ services/ServiceForm.tsx:39]: AddServiceToUserForm

File: src/components/users/UserContacts.tsx [hunk 3]
```diff
@@ -49,8 +48,7 @@ export const UserContacts = ({ user, userServices }: UserContactsProps) => {
                 ))}
             </StyledServicesList>
 
-            {/* TODO: implement AddServiceToUser */}
-            <InlineTrigger icon={<IconPlusCircleOutline noWrap size="s" />} text={tr('Add link')} disabled />
+            <ServiceForm user={user} />
```
Comment (tsumo) [inline src/components/users/UserContacts.tsx:51]: We only need userId really, not the whole user

File: src/components/users/UserPage.tsx [hunk 4]
```diff
@@ -112,6 +112,7 @@ const StyledTeamsLink = styled(Link)`
 export const UserPage = () => {
     const router = useRouter();
     const { userId } = router.query;
+
```
Comment (tsumo) [inline src/components/users/UserPage.tsx:115]: Unnecessary changes in this file

File: src/trpc/router/index.ts [hunk 5]
```diff
@@ -3,11 +3,13 @@ import { router } from '../trpcBackend';
 import { groupRouter } from './group-router';
 import { userRouter } from './user-router';
 import { feedbackRouter } from './feedback-router';
+import { servicesRouter } from './services-router';
 
 export const trpcRouter = router({
     user: userRouter,
     group: groupRouter,
     feedback: feedbackRouter,
+    services: servicesRouter,
```
Comment (tsumo) [inline src/trpc/router/index.ts:12]: Singular form - `service: serviceRouter`

File: src/trpc/router/services-router.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import { userServicesMethods } from '../../modules/services.method';
+import { createServicesSchema, getServiceListSchema } from '../../modules/services.schemas';
+import { protectedProcedure, router } from '../trpcBackend';
+
+export const servicesRouter = router({
```
Comment (tsumo) [inline src/trpc/router/services-router.ts:5]: `serviceRouter`

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/services/AddServiceToUserForm.tsx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+`;
+
+const StyledTableCell = styled(TableCell)`
+    flex-wrap: nowrap;
+    display: flex;
+    align-items: center;
+`;
+
+interface ServicesFormProps {
+    user?: User;
```
Comment (tsumo) [inline src/components/services/AddServiceToUserForm.tsx:36]: Don't need user

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]
cool

**Pull request merged**